screenpipe-app-tauri/src-tauri/src/sidecar.rs

```diff
@@ -369,11 +370,12 @@ fn spawn_sidecar(app: &tauri::AppHandle) -> Result<CommandChild, String> {
     Ok(child)
 }
 pub struct SidecarManager {
-    child: Option<CommandChild>,
+    pub child: Option<CommandChild>,
     last_restart: Instant,
     restart_interval: Arc<Mutex<Duration>>,
     restart_task: Option<JoinHandle<()>>,
     dev_mode: Arc<Mutex<bool>>,
+    pub is_recording: bool,
```

Comment:
this variable is not necessary, we just want to allow user to start or stop the sidecar with less friction than now (atm they need to go in screenpipe app and click on stop/start) we want to save them a few clicks/seconds on this